corehq/apps/app_manager/templates/app_manager/simprints_enrolment_form.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
+	<h:head>
+		<h:title>{{ form_name }}</h:title>
+		<model>
+			<instance>
+				<data xmlns:jrm="http://dev.commcarehq.org/jr/xforms" xmlns="http://openrosa.org/formdesigner/{{ xmlns_uuid }}" uiVersion="1" version="1" name="{{ form_name }}">
+					<name />
+                    <simprintsEnrol />
+                    <simprintsId />
+                    <rightIndex />
+                    <rightThumb />
+                    <leftIndex />
+                    <leftThumb />
+				</data>
```

Comment:
nit: the indentation of this file makes all of this a bit hard to read

Reply:
that's a bit strange. i think that's github...? is see this 
![screen shot 2019-03-01 at 1 45 58 pm](https://user-images.githubusercontent.com/716573/53662404-6a392880-3c28-11e9-8b97-c5bdbee43644.png)

Reply:
![screenshot from 2019-03-01 14-54-02](https://user-images.githubusercontent.com/146896/53662808-e1bf8580-3c31-11e9-85e7-160bffeb462b.png)

You have a mixture of tabs and spaces here.